docs/topics/multiplatform/multiplatform-native-artifacts.md

```diff
@@ -0,0 +1,334 @@
+
+The simplest version requires the `target` (or `targets`) parameter for the selected build type. Currently, 
+two build types are available: 
+
+* `DEBUG` – produces a non-optimized binary with debug information 
+* `RELEASE` – produces an optimized binary without debug information
+
+You can specify for which build types to create binaries in the `modes` parameter. The default value includes both `DEBUG`
+and `RELEASE` executable binaries:
+
+<tabs group="build-script">
+<tab title="Kotlin" group-key="kotlin">
+
+```kotlin
+kotlinArtifacts {
+    Native.Library {
+        target = iosX64 // Define your target instead
+        modes(DEBUG, RELEASE)
+        // Binary configuration
+    }
+}
+```
+
+</tab>
+<tab title="Groovy" group-key="groovy">
+
+```groovy
+kotlinArtifacts {
+    it.native.Library {
+        target = iosX64 // Define your target instead
+        modes(DEBUG, RELEASE)
+        // Binary configuration
+    }
+}
+```
+
+</tab>
+</tabs>
+
+You can also declare binaries with custom names:
+
+<tabs group="build-script">
+<tab title="Kotlin" group-key="kotlin">
+
+```kotlin
+kotlinArtifacts {
+    Native.Library("mylib") {
+        // Binary configuration
+    }
+}
+```
+
+</tab>
+<tab title="Groovy" group-key="groovy">
+
+```groovy
+kotlinArtifacts {
+    it.native.Library("mylib") {
+        // Binary configuration
+    }
+}
+```
+
+</tab>
+</tabs>
+
+The argument sets a name prefix, which is the default name for the binary file. For example, for Windows the code 
+produces the `mylib.dll` file.
+
+## Configure binaries
+
+For the binary configuration, the following common parameters are available:
+
+| **Name**        | **Description**                                                                                                                 |
+|-----------------|---------------------------------------------------------------------------------------------------------------------------------|
+| `isStatic`      | Optional linking type that defines the library type. By default, it's `false` and the library is dynamic.                       |
+| `modes`         | Optional build types, `DEBUG` and `RELEASE`.                                                                                    |
+| `kotlinOptions` | Optional compiler options applied to the compilation. See the list of available [compiler options](gradle.md#compiler-options). |
+| `addModule`     | In addition to the current module, you can add other modules to the resulting artifact.                                         |
+| `setModules`    | You can override the list of all modules that will be added to the resulting artifact.                                          |
+
+### Libraries and frameworks
+
+When building an Objective-C framework or a native library (shared or static), you may need to pack not just the classes
+of the current project, but also the classes of its dependencies into a single entity and export all these modules to it.
```

Comment:
`but also the classes of its dependencies` -> `but also the classes of any other multiplatform module`